server/main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"log"
 
-	"./lib"
+	"github.com/umegaya/channer/server/lib"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/lib (can't find import: github.com/umegaya/channer/server/lib)